app/controllers/register_api/applications_controller.rb

```diff
@@ -2,6 +2,29 @@ module RegisterAPI
   class ApplicationsController < ActionController::API
     include ServiceAPIUserAuthentication
 
-    def index; end
+    def index
+      application_choices = get_recruited_applications(
+        recruitment_cycle_year: recruitment_cycle_year_param,
+      )
+      render json: { data: MultipleApplicationsPresenter.new(application_choices, api: RegisterAPI).as_json }
+    end
+
+  private
+
+    def get_recruited_applications(recruitment_cycle_year:)
+      ApplicationChoice.includes([:application_form])
+        .where(application_forms: { recruitment_cycle_year: recruitment_cycle_year }).references(:application_forms)
+        .where(status: ApplicationStateChange::STATES_VISIBLE_TO_REGISTER)
```

Comment:
I had a look through the state changes and I don't think we need to search for the status, as the `recruited_at` is only unset if we `reinstate_pending_conditions`, but will be set on all the other states. @duncanjbrown can you confirm?